### PR TITLE
Added sounds for unlocking achievements.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Unlocking an achievement now plays a sound. Which sound is played depends on the associated preference of the player, found in the Sound category of the game preferences UI.
The current options are a [glockenspiel ping](https://freesound.org/people/FunWithSound/sounds/456965/), a [beeps jingle](https://freesound.org/people/Eponn/sounds/619838/) and a ["tada!" fanfare](https://freesound.org/people/plasterbrain/sounds/397355/), with the obvious fourth option to not play a sound at all.

All sounds are from Freesound and are public domain. As such, attributions are not required. The sounds have been also converted to .OGG mono 44.1 Khz in accordance to the standards.

## Why It's Good For The Game
Enhancing the player feedback for unlocking an achievement with choosable sounds.

## Changelog

:cl:
sound: Unlocking an achievement now plays a sound by default. You can change it in the Sound category of the game preferences.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
i always think its a nice courtesy to just slap an attributional link in a text file near where the audio file is because as ss14 and other projects pilfer all of our assets it'll strip the information you've provided about its source since it's more than likely they won't look at the associated commit to that file and paste it over and assume it was made in-house with cc 3.0 when it wasn't really...

if that doesn't make sense i just think it's neat because we always thought a certain sound was made by goon post-r4407 and had it in the goon folder when it turns out it was CC 0 all along (#64291)

**Comment by Wallemations** [comment]
Boutta add a downstream PR that just adds this as another option

[https://youtu.be/Vribwyvnw3c?si=s8JDScupi3us6I2e](https://youtu.be/Vribwyvnw3c?si=s8JDScupi3us6I2e)

File: code/__DEFINES/achievements.dm [hunk 1]
```diff
@@ -11,6 +11,12 @@
 ///the priority of the achievements score. NO achievement should have a priority equal or lower than this.
 #define AWARD_PRIORITY_LAST 0
 
+/// preferences for the sound played when unlocking an achievement
+#define CHEEVO_SOUND_TADA "Tada Fanfare"
+#define CHEEVO_SOUND_JINGLE "Beeps Jingle"
+#define CHEEVO_SOUND_PING "Success Ping"
+#define CHEEVO_SOUND_OFF "Disabled"
```
Comment (Time-Green) [inline code/__DEFINES/achievements.dm:18]: I’ll allow it, once…

Comment (Ghommie) [reply code/__DEFINES/achievements.dm:18]: ?

Comment (Time-Green) [reply code/__DEFINES/achievements.dm:18]: You applied your comments awkwardly, autodoccing the top one with just a general remark. But it’s so small and the rest is fine so I usually dont bother

**Pull request merged**